geomesa-kafka/geomesa-kafka-10-datastore/src/main/scala/org/locationtech/geomesa/kafka10/consumer/KafkaConsumerFactory.scala

```diff
@@ -0,0 +1,91 @@
+/***********************************************************************
+* Copyright (c) 2013-2016 Commonwealth Computer Research, Inc.
+* All rights reserved. This program and the accompanying materials
+* are made available under the terms of the Apache License, Version 2.0
+* which accompanies this distribution and is available at
+* http://www.opensource.org/licenses/apache2.0.php.
+*************************************************************************/
+
+package org.locationtech.geomesa.kafka10.consumer
+
+import java.util.Properties
+
+import kafka.consumer._
+import kafka.serializer.{Decoder, DefaultDecoder}
+import org.apache.commons.lang3.RandomStringUtils
+import org.locationtech.geomesa.kafka10.consumer.offsets.OffsetManager
+
+import scala.collection.JavaConverters._
+import scala.collection.Seq
+import scala.language.implicitConversions
+
+/** @param brokers a comma separated list of broker host names and ports
+  * @param zookeepers the zookeeper connection string
+  */
+class KafkaConsumerFactory(brokers: String, zookeepers: String) {
+
+  import KafkaConsumerFactory._
+
+  private val props = {
+    val groupId = RandomStringUtils.randomAlphanumeric(5)
+
+    val props = new Properties()
+    props.put("zookeeper.connect", zookeepers)
+    props.put("bootstrap.servers", brokers)
```

Comment:
Is there a particular reason we're hardcoding the brokers here but using ``KafkaUtils0{8,9}.brokerParam`` in the others?